src/toil_vg/vg_sim.py

```diff
@@ -167,8 +167,10 @@ def run_sim_chunk(job, context, gam, seed, xg_file_id, xg_annot_file_id, chunk_i
 
         # turn the annotated gam json into truth positions, as separate command since
         # we're going to use a different docker container.  (Note, would be nice to
-        # avoid writing the json to disk)        
-        jq_cmd = ['jq', '-c', '-r', '[ .name, .refpos[0].name, .refpos[0].offset ] | @tsv',
+        # avoid writing the json to disk)
+        # note: in the following, we are writing the read name as the first column,
+        # then a path-name, path-offset in each successive pair of columns
+        jq_cmd = ['jq', '-c', '-r', '[ .name, (.refpos[] | .name, .offset) ] | @tsv',
```

Comment:
https://github.com/vgteam/vg/pull/1179 is running into trouble here; there's no check for an empty `.refpos` and so the `jq` command will now crash if it encounters one.

It would also be good if the simulator didn't generate reads with empty refpos annotations.